docs/faq.md

```diff
@@ -1,5 +1,23 @@
 # Frequently Asked Questions
 
+## Volume Attachment Capacity Issues
+
+There's a known issue where a mismatch between the reported and actual attachment capacity on nodes can result in scheduling errors and stuck workloads. This commonly occurs when volume slots are consumed after the driver starts up, which results in `kube-scheduler` assigning stateful pods to nodes lacking the necessary capacity to support volume attachments.
```

Comment:
Would it be useful to add something like 

this results in volumes stuck in the attaching state until a slot is freed up.

Reply:
It would be helpful - added in latest rev.